File: server/src/main/kotlin/server/server/models/WorkingTime.java
package server.server.models;

import jakarta.persistence.*;
import lombok.*;

@Entity
@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
@Builder
public class WorkingTime {
    @Id
    private Long id;

    @OneToOne(cascade = CascadeType.ALL)
    @MapsId
    @JoinColumn(name ="seller_id")
    private Seller seller;

    private String monday;
    private String tuesday;
    private String wednesday;
    private String thursday;
    private String friday;
    private String saturday;
    private String sunday;

}

File: data simulator/models/workingTime.py
class WorkingTime:
    def __init__(self):
        self.monday = None
        self.tuesday = None
        self.wednesday = None
        self.thursday = None
        self.friday = None
        self.saturday = None
        self.sunday = None
        
    def __str__(self):
        return f"""
        {{
            "monday": "{self.monday}", 
            "tuesday": "{self.tuesday}", 
            "wednesday": "{self.wednesday}", 
            "thursday": "{self.thursday}", 
            "friday": "{self.friday}",
            "saturday": "{self.saturday}",
            "sunday": "{self.sunday}"
        }}
    """
        